File: solutions/vbnet/leap/1/Leap.vb
Public Module Leap
    Public Function IsLeapYear(ByVal year As Integer) As Boolean
        If year Mod 4 = 0 And (year Mod 100 <> 0 Or year Mod 400 = 0) Then
            Return True
        End If
        Return False
    End Function
End Module

File: solutions/julia/leap/1/leap.jl
"""
    is_leap_year(year)

Return `true` if `year` is a leap year in the gregorian calendar.

"""
function is_leap_year(year)
    if mod(year,4) == 0 && (mod(year,100) != 0 || mod(year,400) == 0)
        return true
    end
    return false
end

